### PR TITLE
refactor: migrate queries in the graphql folder: Plugins

### PR DESCRIPTION
**Pull request by theodu**

### CODE REVIEW

File: src/kili/graphql/operations/plugin/queries.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+from kili.helpers import format_result, fragment_builder
+from kili.types import Plugin
+
+
+class PluginLogsWhere(NamedTuple):
+    """
+    Tuple to be passed to the PluginQuery to restrict query
+    """
+
+    # pylint: disable=too-many-arguments,too-many-locals,too-many-instance-attributes
```
Comment (PierreLeveau) [inline src/kili/graphql/operations/plugin/queries.py:19]: rm ?

**Comment by theodu** [comment]
This is part of the big project to migrate all queries in a dedicated abstracted folder `graphql`.
The Plugin queries were implemented in a different way than other queries so it does not fully adapt to the GraphQLQuery class but we moved it anyway to finish the project

File: src/kili/graphql/operations/plugin/queries.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+    GQL_GET_PLUGIN_LOGS = """
+    query getPluginLogs(
+        $projectId: ID!
+        $pluginName: String!
+        $createdAt: DateTime
+        $limit: Int
+        $skip: Int
+    ) {
+    data: getPluginLogs(
```
Comment (PierreLeveau) [inline src/kili/graphql/operations/plugin/queries.py:53]: indentation can be improved

**Comment by theodu** [comment]
to be tested before to merge

File: src/kili/graphql/operations/plugin/queries.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    def list(self, fields: List[str]):
+        """
+        List plugins
+        """
+        fragment = fragment_builder(fields, self.FRAGMENT_TYPE)
+        query = self.gql_list_plugins(fragment)
+        result = self.client.execute(query)
+        return format_result("data", result, self.FORMAT_TYPE)
+
+    def logs(self, where: PluginLogsWhere, options: QueryOptions):
```
Comment (PierreLeveau) [inline src/kili/graphql/operations/plugin/queries.py:98]: rename to get_logs

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

**Comment by theodu** [comment]
@PierreLeveau since your last review, I:
- fixed the ident on all queries file
- moved the fragement_builder function inside the GraphQLQuery class

**Review by Jonas1312 — APPROVED** [review]

**Review by PierreLeveau — APPROVED** [review]

**Pull request merged**